azure-pipelines/secure-development-tools.yml

```diff
@@ -9,3 +18,87 @@ steps:
   displayName: 'Run PoliCheck'
   inputs:
     targetType: F
+
+ ### Analyze managed and unmanaged binaries (exe, dll) for security vulnerabilities
+- task: BinSkim@3
+  displayName: Run BinSkim
+  inputs:
+    InputType: 'Basic'
+    Function: 'analyze'
+    AnalyzeTarget: '$(Build.SourcesDirectory)/bin/StreamJsonRpc/$(BuildConfiguration)/netstandard2.0/StreamJsonRpc.dll;$(Build.SourcesDirectory)/bin/StreamJsonRpc/$(BuildConfiguration)/netcoreapp2.1/StreamJsonRpc.dll'
+
+ ### Copy existing Roslyn Analyzer logs to be analyzed
+- task: RoslynAnalyzers@3
+  displayName: Run Roslyn Analyzers.
+  inputs:
+    copyLogsOnly: 'true'
+  continueOnError: True
```

Comment:
We already run the analyzers as part of compilation. Why run them here again? That just seems like it would slow us down and double up on the warnings.

Reply:
This task when using `copyLogsOnly: 'true'` simply copies the Roslyn Analyzer results so that they can be used by `PostAnalysis@1`. This does not run the Roslyn Analyzer again (that would require a full new build which would risk overwriting the assemblies).

Reply:
Why do we need the logs copied? If there were any warnings our build would have broken earlier in the pipeline.

Reply:
Agreed. Removed